### PR TITLE
FLPATH-1162/2

### PR DESCRIPTION
**Pull request by dmartinol**
- Detect changes to individual chart configuration and trigger the corresponding chart release
TODO in FLPATH-1162: 
- Prevent conflicting changes
- Docs gh-pages

### CODE REVIEW

File: .github/workflows/workflows.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Release Helm chart for greeting workflow
```
Comment (masayag) [inline .github/workflows/workflows.yaml:1]: s/greeting/all-workflows/ (or other name of your choice)

File: .github/workflows/mta.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+name: Release Helm chart for greeting workflow
```
Comment (masayag) [inline .github/workflows/mta.yaml:1]: s/greeting/mta

File: .github/workflows/move2kube.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+name: Release Helm chart for greeting workflow
```
Comment (masayag) [inline .github/workflows/move2kube.yaml:1]: s/greeting/move2kube

**Review by masayag — CHANGES_REQUESTED** [review]

File: .github/workflows/workflows.yaml (hunk 1 above)
Comment (dmartinol) [reply .github/workflows/workflows.yaml:1]: do you prefer the `workflows aggregator` or `workflows umbrella chart`? Our mutual friend has some advice:
> Both "Workflows Aggregator" and "Workflows Umbrella Chart" convey the idea of a Helm chart that installs dependencies (referred to as workflows in this case). However, the term "Umbrella Chart" is more commonly used in the Helm community to describe charts that manage multiple sub-charts or dependencies.

> Therefore, "Workflows Umbrella Chart" might be the more appropriate and recognizable term for your use case. It clearly indicates that the chart serves as a container for multiple workflows or dependencies, much like an umbrella covering various components. However, you can choose whichever name you feel best suits the purpose and context of your chart.

Comment (masayag) [reply .github/workflows/workflows.yaml:1]: "Workflows Umbrella Chart"  sounds good.

**Comment by dmartinol** [comment]
will create new PR sorry

**Pull request closed without merging**